docs/synapse/userguides/storm_ref_data_mod.ipynb

```diff
@@ -502,6 +504,204 @@
+    "\n",
+    "*<query>* **[ <(** *<verb>* **)+ {** *<storm>* **} ]**\n",
+    "\n",
+    ".. NOTE::\n",
+    "  The nodes specified by the Storm expression ( ``{ <storm> }`` ) **must already exist** in the Cortex for the light edges to be created.\n",
+    "\n",
+    "**Examples:**\n",
+    "\n",
+    "*Link the specified FQDN and IPv4 to the media:news node referenced by the Storm expression using a \"refs\" light edge:*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make some nodes\n",
+    "q = '[ media:news=(d41d8cd98f00b204e9800998ecf8427e,) inet:fqdn=woot.com inet:ipv4=1.2.3.4]'\n",
+    "# Make some light edges\n",
+    "q1 = \"inet:fqdn=woot.com inet:ipv4=1.2.3.4 [ <(refs)+ { media:news=a3759709982377809f28fc0555a38193 } ]\"\n",
+    "# Display the syntax\n",
+    "print(q1)\n",
+    "# Run the queries and test\n",
+    "podes = await core.eval(q, num=3, cmdr=False)\n",
+    "podes = await core.eval(q1, num=2, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "*Link the specified media:news node to the set of indicators tagged APT1 (#aka.feye.thr.apt1) using a \"refs\" light edge:*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make some nodes\n",
+    "q = '[ inet:fqdn=newsonet.net inet:fqdn=staycools.net +#aka.feye.thr.apt1 ]'\n",
+    "# Make some light edges\n",
+    "q1 = \"media:news=a3759709982377809f28fc0555a38193 [ +(refs)> { +#aka.feye.thr.apt1 } ]\"\n",
+    "# Display the syntax\n",
+    "print(q1)\n",
+    "# Run the queries and test\n",
+    "podes = await core.eval(q, num=2, cmdr=False)\n",
+    "podes = await core.eval(q1, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "*Link the specified inet:cidr4 netblock to any IP address within that netblock that exists in the Cortex ( as referenced by the Storm expression ) using a \"hasip\" light edge:*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make some nodes\n",
+    "q = '[ inet:cidr4=123.120.96.0/24 inet:ipv4=123.120.96.12 inet:ipv4=123.120.96.53 inet:ipv4=123.120.96.202 ]'\n",
+    "# Make some light edges\n",
+    "q1 = \"inet:cidr4=123.120.96.0/24 [ +(hasip)> { inet:ipv4=123.120.96.0/24 } ]\"\n",
+    "# Display the syntax\n",
+    "print(q1)\n",
+    "# Run the queries and test\n",
+    "podes = await core.eval(q, num=4, cmdr=False)\n",
+    "podes = await core.eval(q1, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "**Usage Notes:**\n",
+    "\n",
+    "- No light edge verbs exist in a Cortex by default; they must be created.\n",
```

Comment:
probably want a note about required permissions to make a light edge

Reply:
This is true for everything though - you need perms to delete a node, set a tag, etc. Would rather leave perms discussion to the getting started / devops docs.